bindata/network/mtu-probe/02-job.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: openshift-network-operator
+  name: mtu-prober
+  annotations:
+    kubernetes.io/description: |
+      This job is run early in the network installation process. It determines the MTU
+      of the default route of a node on the cluster.
+spec:
+  selector: {} #need this to make the apiserver happy :/
+  template:
+    spec:
+      containers:
+      - name: prober
+        image: {{.CNOImage}}
+        command:
+        - /usr/bin/cluster-network-operator
+        - probe-mtu
+        - --namespace=openshift-network-operator
```

Comment:
`namespace` and `name` values should templated

Reply:
agreed.